### ISSUE TITLE
Uncaught TypeError: tc.querySelector(...) is null

### ISSUE THREAD

**Issue by VAggrippino** [VAggrippino/todo_core#4]
```
Uncaught TypeError: tc.querySelector(...) is null
    <anonymous> http://127.0.0.1:5500/js/main.js:23
    <anonymous> http://127.0.0.1:5500/js/main.js:12
    EventListener.handleEvent* http://127.0.0.1:5500/js/main.js:1
main.js:23:30
    <anonymous> http://127.0.0.1:5500/js/main.js:23
    forEach self-hosted:203
    <anonymous> http://127.0.0.1:5500/js/main.js:12
    (Async: EventListener.handleEvent)
    <anonymous> http://127.0.0.1:5500/js/main.js:1
```

https://github.com/VAggrippino/todo_core/blob/66dbdd1f3a7ba0019b99bc4e0dc22ceaeda38b4e/js/main.js#L23

`list.type` becomes either `ol` or `ul`.

Although `.list ol` and `.list ul` are definitely both present in the template content, the `ol` element is empty. It seemed redundant to put a list item there that was identical to the one in the `ul`, but this may be causing or related to the problem.

**Comment by VAggrippino** [comment]
This only seems to happen when refreshing after adding a new list.

**Comment by VAggrippino** [comment]
Fixed by 1e2417a
